free-programming-books-ee.md

```diff
@@ -0,0 +1,64 @@
+### Index
+
+* [C](#c)
+* [C Sharp](#c-sharp)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [SQL](#sql)
+* [WebGL](#webgl)
+* [PHP](#php)
+* [Python](#python)
+* [R](#r)
+
+
+
+
+### C
+
+* [C](https://et.wikibooks.org/wiki/Programmeerimiskeel_C) - Wikiõpikud
+
+
+
+### C Sharp
+
+* [Microsoft Visual Studio Code ja C#](https://digiarhiiv.ut.ee/Ained/Doc/VFailid/CSharp_ja_VS.pdf) - Kalle Remm (PDF)
+
+
+
+
+### Java
+
+* [Programmeerimiskeel Java](https://et.wikibooks.org/wiki/Programmeerimiskeel_Java) - Vikiõpikud
+* [Java õppematerjalid](https://ained.ttu.ee/javadoc/index.html) - TTÜ õppematerjalid
+
+
+### JavaScript
+
+* [JavaScript](http://puhang.tpt.edu.ee/raamatud/JavaScript_konspekt.pdf) - Jüri Puhang (PDF)
+
+
+### SQL
+
+* [SQL](https://www.metshein.com/unit/sql-paringukeel/) Metshein.com
+
+
+### WebGL
+
+* [WebGL](http://www.cs.tlu.ee/teemaderegister/get_file.php?id=351) - Raner Piibur (PDF)
```

Comment:
title seems to be "WebGL’i kasutamine interaktiivsete graafikarakenduste loomiseks veebilehitsejas: õppematerjal"